ui/src/shared/components/MarkdownMonacoEditor.tsx

```diff
@@ -6,7 +6,7 @@ import MonacoEditor from 'react-monaco-editor'
 
 // Utils
 import LANGID from 'src/external/monaco.markdown.syntax'
-import THEME_NAME from 'src/external/monaco.markdown.theme'
+import THEME_NAME from 'src/external/monaco.flux.theme'
```

Comment:
fun fact: monaco doesn't support multiple themes!
https://github.com/microsoft/monaco-editor/issues/338

it's in this PR because i couldn't tell if my `result` context changes were causing the editor to lose reference with monaco because of some rerendering bug or if it's the theme's problem itself. i imported the middleware that handles the tokenizing into our codebase so that we can play with prepending TokenizerState tags with language ids as we start to flesh out theming of markdown vs flux and run into collisions.

A PR should be floating through in the next couple days to harden that theme <> onigasm interface a bit more and include the tomls.